.travis.yml

```diff
@@ -2,32 +2,70 @@ language: python
 sudo: false
 python:
   - "2.7"
+
+cache:
+  directories:
+    - node_modules
+    - $HOME/.cache/pip/
+
+matrix:
+  fast_finish: true
+
 env:
-  - TOX_ENV=flake8
-  - TOX_ENV=docs
-  - TOX_ENV=assets
-  - TOX_ENV=es
-  - TOX_ENV=addons-devhub-editors
-  - TOX_ENV=main
+  - TOX_ENV=flake8 APP=xx
+  - TOX_ENV=docs APP=xx
+  - TOX_ENV=assets APP=xx
+  - TOX_ENV=elasticsearch APP=xx
```

Comment:
tox strips empty environment variables so this is a workaround to have `APP` in tox